packages/backend-deployer/src/cdk_deployer.ts

```diff
@@ -10,6 +10,8 @@ import { BackendIdentifier, DeploymentType } from '@aws-amplify/plugin-types';
 import { BackendLocator, CDKContextKey } from '@aws-amplify/platform-core';
 import { BackendDeployerEnvironmentVariables } from './environment_variables.js';
 
+const relativeAmplifyBackendDir = 'amplify';
```

Comment:
should we derive this from `this.backendLocator.locate()` return value ?
Or perhaps add `backendLocator.locateAmplifyDirectory()` ?

Reply:
backendLocator is already only looking in the amplify directory, but could be good to have `backendLocator.locateAmplifyDirectory()` to avoid tight coupling. Will update that

Reply:
Actually I can just grab the dir name from the backendLocator.locate() return value